src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Symbolic/SymbolicRegexNode.cs

```diff
@@ -440,11 +525,25 @@ internal static SymbolicRegexNode<TSet> And(SymbolicRegexBuilder<TSet> builder,
             return CreateCollection(builder, SymbolicRegexNodeKind.And, conjuncts, SymbolicRegexInfo.And(GetInfos(conjuncts)));
         }
 
+        internal static SymbolicRegexNode<TSet> CreateEffect(SymbolicRegexBuilder<TSet> builder, SymbolicRegexNode<TSet> node, SymbolicRegexNode<TSet> effectNode)
```

Comment:
Can you add comments somewhere defining "effect", what it means to create one, how it relates to other nodes, etc?

Reply:
I added a lengthy comment in SymbolicRegexNodeKind.